pkg/server/test/server.go

```diff
@@ -66,6 +66,8 @@ func RunAllBenchmarks(b *testing.B, ds storage.OpenFGADatastore) {
 }
 
 func RunListObjectsBenchmarks(b *testing.B, ds storage.OpenFGADatastore) {
-	b.Run("BenchmarkListObjectsWithReverseExpand", func(b *testing.B) { BenchmarkListObjectsWithReverseExpand(b, ds) })
-	b.Run("BenchmarkListObjectsWithConcurrentChecks", func(b *testing.B) { BenchmarkListObjectsWithConcurrentChecks(b, ds) })
+	b.Run("BenchmarkListObjectsWithMaxConcurrentReadsEqualToOne", func(b *testing.B) { BenchmarkListObjectsWithMaxConcurrentReadsEqualToOne(b, ds) })
```

Comment:
Are we actually benchmarking performance on these things? It feels like these are more of tests than benchmarks. What are we trying to show with these benchmarks?

Reply:
> Are we actually benchmarking performance on these things?

What do you mean? Yes, the PR build should fail if performance gets worse. (I may need to fine tune the `bench` github action in the future + look into Check benchmarks)

> What are we trying to show with these benchmarks?

Not trying to "show" anything, the goal is for PR builds to fail if the benchmark results get worse. I feel like we would have caught the regression fixed in https://github.com/openfga/openfga/pull/1064 if we had had them 🤔

Reply:
Ok so for example, in this PR:

`BenchmarkOpenFGAServer/BenchmarkMemoryDatastore/BenchmarkListObjectsWithMaxResultsEqualToOne                           2        3015844262 ns/op        17364208 B/op     389812 allocs/op`

On your PR https://github.com/openfga/openfga/pull/1064:

`BenchmarkOpenFGAServer/BenchmarkMemoryDatastore/BenchmarkListObjectsWithMaxResultsEqualToOne                         320          18427689 ns/op         4534970 B/op      90500 allocs/op`

:)